### PR TITLE
[knx] Add support for RGBW represented by HSBType

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Allow lossy conversion from RGBW to HSBType and back instead of using separate items for RGB and W.
Select via DPT 251.60600.

> RGBW (DPT 251.600) can either be converted to HSBType, or be represented two items: a HSBType for RGB and an additional PercentType for W channel.
Default handling for RGBW is to use separate items.
Note that this also requires two frames being sent out separately when these elements are sent to the bus, as the binary representation uses a partially populated KNX frame.  
Alternatively, a single HSB item can be used. Conversion to a single HSBType will loose the exact setting for W, and will reconstruct it when a conversion to RGBW is required.
This option can be selected using the special DPT 251.60600.

### CODE REVIEW

**Comment by holgerfriedrich** [comment]
@genesis81 Sorry this comes way later than expected. It replaces #15773 and comes with the benefit that users can choose between using 2 items to control RGB and W separately or converting to a single HSBType. Would you like to test it?

**Comment by genesis81** [comment]
> @genesis81 Sorry this comes way later than expected. It replaces #15773 and comes with the benefit that users can choose between using 2 items to control RGB and W separately or converting to a single HSBType. Would you like to test it?

Thank you for the support. I'm this week a little bit busy but i try to test and give you a feedback

**Comment by lsiepel** [comment]
> > @genesis81 Sorry this comes way later than expected. It replaces #15773 and comes with the benefit that users can choose between using 2 items to control RGB and W separately or converting to a single HSBType. Would you like to test it?
> 
> Thank you for the support. I'm this week a little bit busy but i try to test and give you a feedback

You had any luck testing the change?

**Comment by genesis81** [comment]
> @genesis81 Sorry this comes way later than expected. It replaces #15773 and comes with the benefit that users can choose between using 2 items to control RGB and W separately or converting to a single HSBType. Would you like to test it?

I have tested and works very well.
Thank you for the support. 

Short video of my test:
https://github.com/openhab/openhab-addons/assets/382387/d8d355fa-a0aa-4c2c-a5c3-6609b03a1b96

**Comment by holgerfriedrich** [comment]
Nice demo, @genesis81, thank you!

@lsiepel Could you go forward and review / merge this one? Thanks!

**Comment by lsiepel** [comment]
> Nice demo, @genesis81, thank you!
> 
> @lsiepel Could you go forward and review / merge this one? Thanks!

LGTM, but I prefer a comment from @kaikreuzer as codeowner before I merge. He is also more familiar to rgbw/hsb conversions discussions from the past, not sure if they apply here.

**Comment by holgerfriedrich** [comment]
@lsiepel Fine for me, let's wait for @kaikreuzer s input. Maybe he has additional comments.
FYI: Since the last big restructuring last year we both are code owners of KNX binding.

**Review by J-N-K — APPROVED** [review]
LGTM

**Comment by holgerfriedrich** [comment]
@lsiepel Could we continue with this one, please? Video and approval by J-N-K should be fine for a merge...

**Comment by lsiepel** [comment]
> @lsiepel Could we continue with this one, please? Video and approval by J-N-K should be fine for a merge...

Yes, had some busy days.

**Pull request merged**
